starknet_devnet/blueprints/rpc.py

```diff
@@ -330,6 +338,76 @@ async def get_events():
     raise NotImplementedError()
 
 
+async def add_invoke_transaction(function_invocation: dict, signature: List[str], max_fee: str, version: str) -> dict:
+    """
+    Submit a new transaction to be added to the chain
+    """
+    invoke_function = InvokeFunction(
+        contract_address=int(function_invocation["contract_address"], 16),
+        entry_point_selector=int(function_invocation["entry_point_selector"], 16),
+        calldata=[int(data, 16) for data in function_invocation["calldata"]],
+        max_fee=int(max_fee, 16),
+        version=int(version, 16),
+        signature=[int(data, 16) for data in signature],
+    )
+
+    _, transaction_hash, _ = await state.starknet_wrapper.invoke(invoke_function=invoke_function)
+    return RpcInvokeTransactionResult(
+        transaction_hash=rpc_felt(transaction_hash),
+    )
+
+
+async def add_declare_transaction(contract_class: RpcContractClass, version: str) -> dict:
+    """
+    Submit a new class declaration transaction
+    """
+    try:
+        decompressed_program = decompress_program({"contract_class": contract_class}, False)["contract_class"]
+        contract = ContractClass.loads(json.dumps(decompressed_program, sort_keys=True))
```

Comment:
Also, why `sort_keys=True`? And isn't it simpler to do `ContractClass.load(decompressed_program)` - so `.load` instead of `.loads`?